File: 2017_18/coursework/spike_trains/load.py

def load_data(filename,T):

    data_array = [T(line.strip()) for line in open(filename, 'r')]

    return data_array


#spikes=[int(x) for x in load_data("rho.dat")]
spikes=load_data("rho.dat",int)

print(len(spikes))
print(spikes[0:5])

#stimulus=[float(x) for x in load_data("stim.dat")]
stimulus=load_data("stim.dat",float)

print(len(stimulus))
print(stimulus[0:5])

File: 2017_18/coursework/spike_trains/load.jl

function load_data(filename,T)

    file=open(filename)

    data_array=readdlm(file,T)

    data_array

end

spikes=load_data("rho.dat",Int64)

println(length(spikes))
println(spikes[1:5])

stimulus=load_data("stim.dat",Float64)

println(length(stimulus))
println(stimulus[1:5])
